.circleci/config.yml

```diff
@@ -680,50 +675,51 @@ workflows:
   main-workflow:
     jobs:
       - build-job:
-          name: "Xcode_12.5.1_iOS_14.5_Debug"
-          xcode: "12.5.1"
-          iOS: "14.5"
-          device: "iPhone 12 Pro Max"
+          name: "Xcode_13.2.1_iOS_15.2_Debug"
+          xcode: "13.2.1"
+          iOS: "15.2"
+          device: "iPhone 13 Pro Max"
           context: Slack Orb
       - build-job:
-          name: "Xcode_12.5.1_iOS_14.5_SPM_Debug"
-          xcode: "12.5.1"
-          iOS: "14.5"
-          device: "iPhone 12 Pro Max"
+          name: "Xcode_13.2.1_iOS_15.2_SPM_Debug"
+          xcode: "13.2.1"
+          iOS: "15.2"
+          device: "iPhone 13 Pro Max"
           spm: true
           codecoverage: false
           context: Slack Orb
       - build-job:
-          name: "Xcode_12.5.1_iOS_14.5_Release"
-          xcode: "12.5.1"
-          iOS: "14.5"
-          device: "iPhone 12 Pro Max"
+          name: "Xcode_13.2.1_iOS_15.2_Release"
+          xcode: "13.2.1"
+          iOS: "15.2"
+          device: "iPhone 13 Pro Max"
           context: Slack Orb
           configuration: "Release"
       - build-job:
-          name: "Xcode_12.5.1_iOS_14.5_SPM_Release"
-          xcode: "12.5.1"
-          iOS: "14.5"
-          device: "iPhone 12 Pro Max"
+          name: "Xcode_13.2.1_iOS_15.2_SPM_Release"
+          xcode: "13.2.1"
+          iOS: "15.2"
+          device: "iPhone 13 Pro Max"
           spm: true
           codecoverage: false
           context: Slack Orb
           configuration: "Release"
       - pod-job:
-          name: "Xcode_12.5_iOS_14.5_CP_install"
+          name: "Xcode_13.2.1_iOS_15.2_CP_install"
           update: false
-          xcode: "12.5.0"
-          iOS: "14.5"
+          xcode: "13.2.1"
+          iOS: "15.2"
           archive: true
           context: Slack Orb
       - pod-job:
-          name: "Xcode_12.5_iOS_14.5_CP_update"
+          name: "Xcode_13.2.1_iOS_15.2_CP_update"
           update: true
-          xcode: "12.5.0"
-          iOS: "14.5"
+          xcode: "13.2.1"
+          iOS: "15.2"
           lint: true
           context: Slack Orb
-      - xcode-12-examples:
+      - build-example:
+          name: "Build Example"
           context: Slack Orb
       - spm-test-job:
           name: "swift test; Xcode 12.5; iOS 14.5"
```

Comment:
Looks like we’re only retaining Xcode 12 and iOS 13–14 tests of MapboxCoreNavigation; is that sufficient coverage for these older versions? What if we break the build in MapboxNavigation on Xcode 12? If we intend to drop support for Xcode 12 – which would make some sense given the impending App Store requirement – then the documentation and changelog should be updated to reflect that change.

Reply:
`spm-test-job` [runs snapshot tests as well](https://app.circleci.com/pipelines/github/mapbox/mapbox-navigation-ios/7325/workflows/1c246565-4cbb-4b06-8089-281988cbde84/jobs/61431), so we still test everything on Xcode 12, except for the Carthage integration.

We also have "extended workflow" in this config, which runs even on Xcode 11